frontend/src/main/scala/bloop/testing/TestSuiteEvent.scala

```diff
@@ -105,13 +116,18 @@ class LoggingEventHandler(logger: Logger) extends TestSuiteEventHandler {
       logger.info(s"All $suitesPassed test suites passed.")
     } else {
       val metrics =
-        List(suitesPassed -> "passed", suitesFailed.length -> "failed", suitesAborted -> "aborted")
+        List(suitesPassed -> "passed", testsFailedBySuite.size -> "failed", suitesAborted -> "aborted")
       logger.info(formatMetrics(metrics))
 
-      if (suitesFailed.nonEmpty) {
+      if (testsFailedBySuite.nonEmpty) {
         logger.info("")
         logger.info("Failed:")
-        suitesFailed.foreach(suite => logger.info(s"- $suite"))
+        testsFailedBySuite.foreach{case(suiteName, tests) =>
+          logger.info(s"- $suiteName")
+          tests.foreach{ case(testName, failure) =>
+            logger.info(s"$testName - ${failure.map(_.getMessage)}")
```

Comment:
We might want to add some indentation here to make clear the test name it's a child of the suite name, and even play around a little bit with colored output to make it easier to read. Feel free to also change the whole way we print the summary at the end.
  
We can probably draw some inspiration from great testing libraries in other languages such as avajs https://github.com/avajs/ava -- Nobody has done any research on what would be the best way to tell the user an error fail, so this would be a good opportunity to do so :smile: